src/FunManifolds.jl

```diff
@@ -91,23 +94,25 @@ end
 
 function concretize_tols(M::Manifold, x1, x2; reltol = nothing, abstol = nothing)
     rtol = if reltol === nothing
-        rtoldefault(M.M, x1, x2)
+        rtoldefault(M, x1, x2)
     else
         reltol
     end
 
     atol = if abstol === nothing
-        atoldefault(M.M, x1, x2)
+        atoldefault(M, x1, x2)
     else
         abstol
     end
 
     return (rtol, atol)
 end
 
+include("DCurves.jl")
```

Comment:
Maybe `DiscretizedCurves` is a better name?

Reply:
Yes, I'll most likely change it at a later stage of porting.